code/game/objects/items/spear.dm

```diff
@@ -10,6 +10,7 @@
 	slot_flags = ITEM_SLOT_BACK
 	throwforce = 20
 	throw_speed = 4
+	demolition_mod = 0.75
```

Comment:
does demolition_mod also applies to thrown impacts? if so, making it impossible to (normal) spear open lockers is a pretty significant balance chance, no?

Reply:
It doesn't change throw force